test-prow-e2e.sh

```diff
@@ -2,6 +2,90 @@
 
 set -eExuo pipefail
 
+function installOperatorSDK {
+    export ARCH=$(case $(uname -m) in x86_64) echo -n amd64 ;; aarch64) echo -n arm64 ;; *) echo -n $(uname -m) ;; esac)
+    export OS=$(uname | awk '{print tolower($0)}')
+    export OPERATOR_SDK_DL_URL=https://github.com/operator-framework/operator-sdk/releases/download/v1.20.0
+    curl -LO ${OPERATOR_SDK_DL_URL}/operator-sdk_${OS}_${ARCH}
+    chmod +x operator-sdk_${OS}_${ARCH} && sudo mv operator-sdk_${OS}_${ARCH} /usr/local/bin/operator-sdk
+}
+
+function installOperator {
+    export TAG=ci
+    export IMG=quay.io/rexagod/mcg-osd-deployer:$TAG
+    export BUNDLE_IMG=quay.io/rexagod/mcg-osd-deployer:bundle-$TAG
```

Comment:
I think here we need to provide mcg-osd repo url,

Reply:
That was my first thought as well, but I saw that the `bundle`, `index` and `prometheus` images were last updated on 27th April. Should I still use https://quay.io/repository/osd-addons/mcg-osd?

Reply:
I agree we should use mcg-osd. Let's update the Images if required but we should use that only.

Reply:
@naveenpaul1 Do you have push access to this? I haven't heard back from the MT-SRE team on my request to grant me those rights, but as long as someone from our team can push images for testing, I can go ahead and make this change, PLMK, thanks.

Reply:
I have push access to `osd-addons/mcg-osd` I can push the image

Reply:
/LGTM